pkg/k8s/apis/cilium.io/v2alpha1/flowlog.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package v2alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:onlyVerbs=get,list,watch
```

Comment:
Why don't we want clients to create, patch, update, etc?

Reply:
They can be generated when (and if) they are needed.

Reply:
Why? It seems like this is generated code so it doesn't cost us anything to add it right now and there's no additional maintenance.

If we wait until we need it later, somebody will have to do extra work and possibly cherry picks.

Reply:
`patch` and `update` are unsupported, so I added `create` only.